update/genesis/import.go

```diff
@@ -238,7 +238,8 @@ func (si *stateImport) getTrie(shardID uint32, accType Type) (data.Trie, error)
 		trieStorageManager = si.trieStorageManagers[triesFactory.PeerAccountTrie]
 	}
 
-	trieForShard, err := trie.NewTrie(trieStorageManager, si.marshalizer, si.hasher)
+	maxTrieLevelInMemory := uint(5)
```

Comment:
use the inputted trie level - or make a constant.

Reply:
Done.